web/src/features/fbaCalculator/components/WindSpeedCell.tsx

```diff
@@ -27,7 +27,7 @@ export interface WindSpeedCellProps {
   rowId: number
 }
 
-const adjustedTheme = createTheme({
+export const adjustedTheme = createTheme({
```

Comment:
Can we move this to a shared place, maybe where the other fire table themes are (`web/src/app/theme.ts`)?

Reply:
Done in https://github.com/bcgov/wps/pull/4407/commits/149b8a50e8291b98e6a12fe9f57677c64a451065